src/js/controllers/modals/search.js

```diff
@@ -50,8 +52,15 @@ angular.module('copayApp.controllers').controller('searchController', function($
         return lodash.includes(tx.searcheableString, search.toLowerCase());
       });
 
+      if (search && (search.indexOf('bitcoincash:') >= 0 || search[0] === 'C' || search[0] === 'H' || search[0] === 'p' || search[0] === 'q') && search.replace('bitcoincash:', '').length === 42) {
```

Comment:
We should recognize the BTC address as well.

Reply:
Also legacy addresses for BCH